uniffi_bindgen/src/bindings/python/templates/BytesHelper.py

```diff
@@ -8,5 +8,9 @@ def read(buf):
 
     @staticmethod
     def write(value, buf):
+        try:
```

Comment:
Why not `isinstance()` here? I assume it's because you want to support anything exposing the buffer protocol - so why do bytes get "anything bytes-like" but strings aren't "anything string-like"?